app/furniture/tobias/payout.rb

```diff
@@ -0,0 +1,19 @@
+class Tobias
+  class Payout < ApplicationRecord
+    self.table_name = "tobias_payouts"
+
+    belongs_to :trust
+    has_many :beneficiaries, through: :trust
+    has_many :payments
+
+    monetize :payout_amount_cents
```

Comment:
The table is named `payout`, so naming this `payout_amount_cents` feels unnecessarily redundant.